typescript/infrastructure/ci.yaml

```diff
@@ -289,7 +289,7 @@ Resources:
 #          - Effect: Allow
 #            Action:
 #              - dynamodb:*
-#            Resource: !Sub "arn:aws:apigateway:${AWS::Region}::/*"
+#            Resource: "*"
```

Comment:
Why wasn't it working?  If anything this should be locked down better rather than made more open.